src/credentials/GroupDataProviderImpl.cpp

```diff
@@ -1558,10 +1723,48 @@ CHIP_ERROR GroupDataProviderImpl::RemoveFabric(chip::FabricIndex fabric_index)
 CHIP_ERROR GroupDataProviderImpl::Decrypt(PacketHeader packetHeader, PayloadHeader & payloadHeader,
                                           System::PacketBufferHandle & msg)
 {
-    (void) packetHeader;
-    (void) payloadHeader;
-    (void) msg;
-    return CHIP_NO_ERROR;
+    chip::GroupId group_id = packetHeader.GetDestinationGroupId().Value();
```

Comment:
```suggestion
    GroupId group_id = packetHeader.GetDestinationGroupId().Value();
```
and same through all the changes in this file....  Please don't add unnecessary "chip::".